src/main/java/org/jenkinsci/plugins/gitclient/JGitAPIImpl.java

```diff
@@ -765,10 +765,10 @@ public void deleteRef(String refName) throws GitException, InterruptedException
 	    refPrefix = refPrefix.replace(' ', '_');
 	}
 	try (Repository repo = getRepository()) {
-	    Map<String, Ref> refList = repo.getRefDatabase().getRefs(refPrefix);
+	    List<Ref> refList = repo.getRefDatabase().getRefsByPrefix(refPrefix);
```

Comment:
The comment below is out-of-sync, still about the "key set".

Reply:
Thanks.  That comment is incorrect now.  That code is now just converting a `List<Ref>` into a `Set<String>`.  I'll delete the incorrect comment.